app/db/__init__.py

```
@@ -0,0 +1,3 @@
+"""Database utilities for CryptoUniverse tests and local setup."""
+
+from .seeds import seed_core_data  # noqa: F401
```

Ruff (0.14.3)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)